utils/outfit-serial-list-generator.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/python
+# check_content_style.py
+# Copyright (c) 2024 by Daeridanii
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+categories = [
+    "Guns",
+    "Turrets",
+    "Secondary Weapons",
+    "Ammunition",
+    "Systems",
+    "Power",
+    "Engines",
+    "Hand to Hand",
+    "Unique",
+    "Licenses"
+]
+
+turrets_series = [
+    "Turrets",
+    "Anti-Missile",
+]
+
+systems_series = [
+    "$Drives", # The $ creates a single category shared by all species.
+    "Shields",
+    "Repair",
+    "Cooling",
+    "Scanners",
+    "Jammers",
+    "Ramscoops",
+    "Special Systems",
+    "$Fuel",
+    "$Passenger",
+    "%Utility", # The % lists this with index 999, so it comes after all other subseries.
+]
+
+power_series = [
+    "Generators",
+    "Batteries",
+    "Solar",
+]
+
+engines_series = [
+    "Engines",
```

Comment:
Don't suppose it would be worth splitting this into Thrusters, Steering, and Reverse Thrusters?
My guess would be that's not a great idea - where do the X1000 (and Jibauni engines) go, in that case.

Reply:
This is doable but would contradict the existing precedent for ordering human engines, and, in my opinion, worsen user experience when multiple species' engines are present in a single outfitter, as steering/thruster/reverse versions of a single engine size would be split by large intervening space.